server/models/files.js

```diff
@@ -0,0 +1,18 @@
+import mongoose from "mongoose";
+
+const dataSchema = new mongoose.model(
+  "Files",
+  new mongoose.Schema({
+    title: {
```

Comment:
Are we just linking to a viz based on the title?

Reply:
Yep, it's based on the visualization's title